v2.1/reference/tools/download.md

```diff
@@ -8,16 +8,24 @@ category: reference
 
 This document collects the available downloads for most officially maintained versions of TiDB enterprise tools.
 
-## TiDB-Binlog and Lightning
+## TiDB Binlog
 
-If you want to download the latest version of [TiDB Binlog](/reference/tidb-binlog-overview.md) or [Lightning](/reference/tools/tidb-lightning/overview.md), directly download the TiDB package, because both TiDB-Binlog and Lightning are included in the TiDB package.
+If you want to download the latest version of [TiDB Binlog](/reference/tidb-binlog-overview.md), directly download the TiDB package, because TiDB Binlog is included in the TiDB package.
 
-In addition, the Kafka version of TiDB-Binlog is also provided.
+In addition, the Kafka version of TiDB Binlog is also provided.
 
 | Package name | OS | Architecture | SHA256 checksum |
 |:---|:---|:---|:---|
-| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB-Binlog, Lightning) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
-| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz) (the Kafka version of TiDB-Binlog) | Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB Binlog) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz) (the Kafka version of TiDB Binlog) | Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+
+## TiDB Lightning
+
+If you want to download the latest version of [TiDB Lightning](/reference/tools/tidb-lightning/overview.md), click the download link in the following table.
+
+| Package name | OS | Architecture |  SHA256 checksum |
+|:---|:---|:---|:---|
+| [tidb-toolkit-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-toolkit-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.sha256) |
```

Comment:
I'm not sure if we should use the "latest" package on the v2.1 doc

Reply:
v2.1 doc should not separate the TiDB Lightning since the lightning binary is still keep with TiDB server package.  
And yes, v2.1 should not use the latest tag.

Reply:
In v2.1 doc, i changed the tag from latest to v2.1.0 and do not separate the TiDB Lightning from the TiDB pkg. commit: https://github.com/pingcap/docs/pull/1337/commits/ec846a705f6722a3e73eb0853310f8c8d6f5d152
In v3.0 doc, i changed the tag from latest to v3.0.0. commit: https://github.com/pingcap/docs/pull/1337/commits/9683868feaf998f02c0b0096da1286c8a57e1681